WordPressShared.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressShared"
-  s.version       = "1.4.0"
+  s.version       = "1.5.0-beta.1"
```

Comment:
Since this is a minor change, should the version be `1.4.1-beta.1`?

Reply:
@ScoutHarris  I followed the SemVer approach, changing the minor version instead of the patch version, since this PR adds new events and is not bug fix related.

Reply:
Gotcha. And thanks for the explanation via Slack. I see what you did there. 😄